Cargo.toml

```diff
@@ -16,7 +16,9 @@ dir = []
 [dependencies]
 bytes = "1.0"
 flate2 = "1.0.1"
-futures = "0.3.1"
```

Comment:
Hmm, after doing a merge, I did a local run of `cargo test`. It doesn't pass! `tests/*` and `examples/*` still reference `futures::`'. I'm surprised because CI passed.

It looks like the `--benches` argument in the CI excludes the other stuff; it should be `--all-targets` instead. I'll fix that now, and then the build errors, assuming they're just trivial renames like the ones here.

Reply:
Oh, sorry for that. I should have tried both.

The futures change wasn't actually needed, it's just too trim down the dependencies.

Reply:
No worries. This is the sort of thing that happens when CI is broken. It's fixed now.